### PR TITLE
IObit.Unlocker version 1.2.0.2

### PR DESCRIPTION
**Pull request by Sky-NiniKo**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56584)

### CODE REVIEW

File: manifests/i/IObit/Unlocker/1.2.0.2/IObit.Unlocker.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+PackageVersion: 1.2.0.2
+PackageLocale: en-US
+Publisher: IObit
+PublisherUrl: https://www.iobit.com/
+PublisherSupportUrl: https://www.iobit.com/onlinefeedback.php
+PrivacyUrl: https://www.iobit.com/privacy.php
+Author: IObit
+PackageName: IObit Unlocker
+PackageUrl: https://www.iobit.com/en/iobit-unlocker.php
+License: © IObit. All rights reserved.
```
Comment (Trenly) [inline manifests/i/IObit/Unlocker/1.2.0.2/IObit.Unlocker.locale.en-US.yaml:14]:
```suggestion
License: Proprietary
```

File: manifests/i/IObit/Unlocker/1.2.0.2/IObit.Unlocker.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+LicenseUrl: https://www.iobit.com/legal.php
```
Comment (Trenly) [inline manifests/i/IObit/Unlocker/1.2.0.2/IObit.Unlocker.locale.en-US.yaml:15]:
```suggestion
LicenseUrl: https://www.iobit.com/eula.php
```

**Review by Trenly — CHANGES_REQUESTED** [review]
A Copyright is not a form of License

**Review by ItzLevvie — APPROVED** [review]

**Comment by Sky-NiniKo** [comment]
Hello, the scan says "Error information: BlockingDetectionFound". From what I understood from the [article](https://docs.microsoft.com/en-us/windows/package-manager/package/binary-validation-errors), the scan thinks it is a virus. The [article](https://docs.microsoft.com/en-us/windows/package-manager/package/binary-validation-errors#understanding-the-installers-scan-test) also says: "Microsoft will first attempt to verify that the flagged software is not a false positive with the antivirus vendors". Do I have to do anything to make this happen? Thank you in advance for your answer. NiniKo.

**Comment by Sky-NiniKo** [comment]
> @wingetbot run

It doesn't change anything. Should I consider that this program is detected as a virus and will never be accepted?

**Comment by Sky-NiniKo** [comment]
@ItzLevvie 
I don't need a recheck, I need an answer:  Should I consider that this program is detected as a virus and will never be accepted?
Sorry if I sound annoyed but that's not my intention I just want to know if we should close this PR.
And thank you very much for your work.

**Comment by ItzLevvie** [comment]
> Should I consider that this program is detected as a virus and will never be accepted?

Yes because it's flagged as either a malware or a PUA by anti-virus software.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/56584#issuecomment-1090246289

**Comment by Sky-NiniKo** [comment]
I close the PR. See [#56584 (comment)](https://github.com/microsoft/winget-pkgs/pull/56584#issuecomment-1090246289) for more explanations.

**Pull request closed without merging**